modules/swagger-codegen/src/main/resources/Java/libraries/jersey2/ApiClient.mustache

```diff
@@ -526,8 +526,8 @@ public class ApiClient {
           serviceException = response.readEntity(ServiceException.class);
           serviceException.setResponseHeaders(responseHeaders);
         } catch (RuntimeException e) {
-          // e.printStackTrace();
-        }
+          throw new ApiException(response.getStatus(),responseHeaders,"Failed to build ServiceException from service response: " + e.getMessage());
```

Comment:
I think it would be more preferable to use the other `ApiException` constructor that passes in the entire exception instead of pulling out the message:

```java
ApiException(String message, Throwable throwable, int code, Map<String, List<String>> responseHeaders, String responseBody)
```